contracts/access/AccessControlDefaultAdminRules.sol

```diff
@@ -222,19 +303,26 @@ abstract contract AccessControlDefaultAdminRules is IAccessControlDefaultAdminRu
         super._revokeRole(role, account);
     }
 
+    /**
+     * @dev Resets the pending default admin and delayed until.
+     */
+    function _resetDefaultAdminDelayChange() private {
+        delete _pendingDefaultAdminDelay;
+        delete _defaultAdminDelayChangeSchedule;
+    }
+
     /**
      * @dev Resets the pending default admin and delayed until.
      */
     function _resetDefaultAdminTransfer() private {
         delete _pendingDefaultAdmin;
-        delete _defaultAdminTransferDelayedUntil;
+        delete _defaultAdminTransferSchedule;
     }
 
     /**
-     * @dev Checks if a {defaultAdminTransferDelayedUntil} has been set and passed.
+     * @dev Checks if a {defaultAdminTransferSchedule} has been set and passed.
      */
-    function _hasDefaultAdminTransferDelayPassed() private view returns (bool) {
-        uint48 delayedUntil = defaultAdminTransferDelayedUntil();
-        return delayedUntil > 0 && delayedUntil < block.timestamp;
+    function _hasScheduledPassed(uint48 schedule) private view returns (bool) {
```

Comment:
There is a typo in `Scheduled`, it should say `Schedule`, I believe.

Reply:
I changed it to just `_hasPassed` [here](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4079/commits/9e2bda997a56d26657d02df81f910af65b723304). I think the parameter is indicative of what has actually passed.

Reply:
Solving since I also needed to split this function in two: `_hasPassed` and `_isSet`, which is ultimately removing the initial concern here. Feel free to reopen if the naming still sounds bad